build.cake

```diff
@@ -41,12 +41,16 @@ Task("Test")
       // Use MSBuild
       MSBuild("./src/XComponent.Functions.Test/XComponent.Functions.Test.csproj",
```

Comment:
Why don't we always use XcBuild ?

Reply:
Good question... 

@fredericcarre  any comments?